### PR TITLE
Frontend CICD fixes

### PR DESCRIPTION
**Pull request by danielogen**
#### Summary of Changes (What does this PR do?)
- Fix cicd frontend 

#### Is this change ready to hit production in its current state?
- [X] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [X] I consider this code done
- [X] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- [ticket_id]()

#### Screenshots (optional)

### CODE REVIEW

File: netmanager/Dockerfile.stage [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
-ENV PATH /usr/src/app/node_modules/.bin:$PATH
-COPY package.json /usr/src/app/package.json
-RUN npm install 
-RUN npm install @react-leaflet/core
-RUN npm install react-scripts@^3.4.3 -g
-COPY . /usr/src/app
-RUN npm run build
+### STAGE 2: Production Environment ###
```
Comment (NoahNsimbe) [inline netmanager/Dockerfile.stage:1]: @danielogen a small typo

File: netmanager/Dockerfile.stage [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
+FROM nginx:1.13.12-alpine
 
+# nginx configuration
+COPY ./nginx.conf /etc/nginx/nginx.conf
 
-# production environment
-FROM nginx:1.13.9-alpine
-COPY --from=builder /usr/src/app/build /usr/share/nginx/html
+# move build folder to nginx
+COPY ./build /usr/share/nginx/html
```
Comment (NoahNsimbe) [inline netmanager/Dockerfile.stage:8]: @danielogen sorry for the late feedback, just realised that there is not command within the Dockerfile that creates the `build` folder.

Comment (NoahNsimbe) [reply netmanager/Dockerfile.stage:8]: Same applies to the [production Dockerfile](https://github.com/airqo-platform/AirQo-frontend/blob/staging/netmanager/Dockerfile.prod).

**Review by NoahNsimbe — APPROVED** [review]
LGTM!

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**